docs/content/preview/yugabyte-platform/_index.md

```diff
@@ -12,10 +12,15 @@ menu:
     weight: 10
```

Comment:
If you add headcontent the text should display under the title

Reply:
I don't really want that.